perfkitbenchmarker/linux_packages/cassandra.py

```diff
@@ -196,7 +271,8 @@ def StartCluster(seed_vm, vms):
   time.sleep(NODE_START_SLEEP)
   for i in xrange(5):
     if not IsRunning(seed_vm):
-      logging.warn('Seed %s: Cassandra not running yet (try %d). Waiting %ds.',
+      logging.warn('Seed %s: Cassandra not running yet (try %d). " \
```

Comment:
I thought I hit a lint error with this line (length), but the reverted version is under 80 characters.